modules/perception/camera/lib/traffic_light/detector/recognition/classify.h

```diff
@@ -24,7 +24,7 @@
 #include "modules/perception/base/traffic_light.h"
 #include "modules/perception/camera/lib/interface/base_traffic_light_detector.h"
 #include "modules/perception/camera/lib/traffic_light/detector/recognition/proto/recognition.pb.h"
-#include "modules/perception/inference/inference.h"
+#include "modules/perception/camera/common/camera_frame.h"
```

Comment:
Why do we need this header, "camera_frame.h"? Besides, header files are better listed in alphabetical order.

Reply:
I found that the header file was missing when compiling before，but now it can be successfully compiled by removing it. I will remove the redundant header files soon.